Artifacts/PgSql/Data/Security/1060-RemoveRedundantBootstrapPermissions.sql

```diff
@@ -0,0 +1,48 @@
+    -- SPDX-License-Identifier: Apache-2.0
+    -- Licensed to the Ed-Fi Alliance under one or more agreements.
+    -- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+    -- See the LICENSE and NOTICES files in the project root for more information.
+
+    DO language plpgsql $$
+    DECLARE
+    claim_Name VARCHAR(250) := 'http://ed-fi.org/ods/identity/claims/domains/educationOrganizations';
+    claimSet_Name VARCHAR(250) := 'Bootstrap Descriptors and EdOrgs';
+    educationOrganizationsResourceClaimId INT;
+    claimSet_Id INT;
+    createAction_Id INT;
+    authorizationStrategy_Id INT;
+
+    BEGIN
+
+    SELECT AuthorizationStrategyId INTO authorizationStrategy_Id FROM dbo.AuthorizationStrategies where DisplayName='No Further Authorization Required';
+    SELECT ActionId INTO createAction_Id FROM dbo.Actions where ActionName='Create';
```

Comment:
Always make SQL keyword UPPER CASE (so... `where` should be `WHERE`).

Reply:
I fixed it . Thanks